analyzers/src/SonarAnalyzer.Common/Rules/Hotspots/RequestsWithExcessiveLengthBase.cs

```diff
@@ -136,6 +144,43 @@ private bool IsEnabled(AnalyzerOptions options)
             return SupportedDiagnostics.Any(d => analyzerConfiguration.IsEnabled(d.Id));
         }
 
+        private void CheckWebConfig(SonarAnalysisContext context, CompilationAnalysisContext c)
+        {
+            foreach (var fullPath in context.GetWebConfig(c))
+            {
+                var webConfig = File.ReadAllText(fullPath);
+                if (webConfig.Contains("<system.web") && XmlHelper.ParseXDocument(webConfig) is { } doc)
```

Comment:
what if it contains only `<system.webServer`?

maybe add tests with only `system.web` and only `system.webServer`

and with attributes which have the sought names, but are not under the correct XML elements

Reply:
what I meant was adding a file
```
<configuration>
  <system.web>
    <httpRuntime maxRequestLength="81920" executionTimeout="3600" /> <!-- Noncompliant {{Make sure the content length limit is safe here.}} -->
  </system.web>
</configuration>
```

and a file

```
<configuration>
  <system.webServer>
    <security>
      <requestFiltering>
        <requestLimits maxAllowedContentLength="83886080" />         <!-- Noncompliant {{Make sure the content length limit is safe here.}} -->
      </requestFiltering>
    </security>
  </system.webServer>
</configuration>
```

I expect this last one to fail because of this `if`.

Reply:
It didn't fail, since <system.web> and <system.webServer> both start with <system.web

Reply:
Ah nice! My bad